### ISSUE TITLE
New fork for IterativeStratification

### ISSUE THREAD

**Issue by crypdick** [scikit-multilearn/scikit-multilearn#219]
Unfortunately, this library seems like abandon-ware. If anyone depends on `IterativeStratification`, I just created an updated version here: https://github.com/crypdick/multilabel-stratifier

Contributions welcome!

**Comment by niedakh** [comment]
Yeah, I didn't have too much time to maintain this library through the pandemic, but if you could provide a PR with the changes I'd gladly merge itd.
